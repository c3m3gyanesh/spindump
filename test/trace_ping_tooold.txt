A ping response comes back too late, Spindump should reject the way too long RTT
