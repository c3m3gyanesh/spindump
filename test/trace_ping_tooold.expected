ICMP 10.0.1.36 <-> 212.16.98.51 29887 at 1571653866918427 new starting packets 0 0 bytes 0 0
